### ISSUE TITLE
Error during setup of submission import using Feed Me

### ISSUE THREAD

**Issue by bavocorteville** [verbb/formie#1485]
### Describe the bug

During the setup of the submission import using Feed-Me, an error occurs.

```
Twig\Error\RuntimeError: Impossible to access an attribute ("name") on a string variable ("id"). in /var/www/html/vendor/verbb/formie/src/templates/integrations/feedme/fields/repeater.html:46
Stack trace:
#0 /var/www/html/vendor/craftcms/cms/src/helpers/Template.php(146): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), 'id', 'name', Array, 'any', false, false)
#1 /var/www/html/storage/runtime/compiled_templates/37/3766ec7843dc5278d8bbe04af5eb3ea2.php(134): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), 'id', 'name', Array)
#2 /var/www/html/vendor/twig/twig/src/Template.php(394): __TwigTemplate_145c560276f6a4fdfe39a7d782b49ce7->doDisplay(Array, Array)
#3 /var/www/html/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#4 /var/www/html/storage/runtime/compiled_templates/f8/f8ec5b9695c5b708f3fd45f1164c43f5.php(209): Twig\Template->display(Array)
#5 /var/www/html/vendor/twig/twig/src/Template.php(394): __TwigTemplate_ce7d4e0d13b4038e7a10ae449097b60a->doDisplay(Array, Array)
#6 /var/www/html/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#7 /var/www/html/storage/runtime/compiled_templates/e9/e9e43b53633d303a365402ef4a162fe3.php(169): Twig\Template->display(Array)
#8 /var/www/html/vendor/twig/twig/src/Template.php(171): __TwigTemplate_bf8bbf369c7a92c233be3ed10e0cecfb->block_content(Array, Array)
#9 /var/www/html/storage/runtime/compiled_templates/0f/0f34ad0db82f85c2ac32a554845b2a25.php(592): Twig\Template->displayBlock('content', Array, Array)
#10 /var/www/html/vendor/twig/twig/src/Template.php(171): __TwigTemplate_849d85b4d8cbfed2edbd0eb7f84f0708->block_main(Array, Array)
#11 /var/www/html/storage/runtime/compiled_templates/0f/0f34ad0db82f85c2ac32a554845b2a25.php(376): Twig\Template->displayBlock('main', Array, Array)
#12 /var/www/html/vendor/twig/twig/src/Template.php(171): __TwigTemplate_849d85b4d8cbfed2edbd0eb7f84f0708->block_body(Array, Array)
#13 /var/www/html/storage/runtime/compiled_templates/4c/4cac69a0c541189ed1ef68101885cd52.php(94): Twig\Template->displayBlock('body', Array, Array)
#14 /var/www/html/vendor/twig/twig/src/Template.php(394): __TwigTemplate_245fc555adb4ccea5234099b0dd6f3c5->doDisplay(Array, Array)
#15 /var/www/html/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#16 /var/www/html/storage/runtime/compiled_templates/e9/e9afce565c8a66c95daf68326e635855.php(56): Twig\Template->display(Array, Array)
#17 /var/www/html/vendor/twig/twig/src/Template.php(394): __TwigTemplate_650f567c07db1ecf9c01c13143bd45b3->doDisplay(Array, Array)
#18 /var/www/html/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#19 /var/www/html/storage/runtime/compiled_templates/0f/0f34ad0db82f85c2ac32a554845b2a25.php(179): Twig\Template->display(Array, Array)
#20 /var/www/html/vendor/twig/twig/src/Template.php(394): __TwigTemplate_849d85b4d8cbfed2edbd0eb7f84f0708->doDisplay(Array, Array)
#21 /var/www/html/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#22 /var/www/html/storage/runtime/compiled_templates/3b/3bcc9194130352c09552e4db4013699a.php(57): Twig\Template->display(Array, Array)
#23 /var/www/html/vendor/twig/twig/src/Template.php(394): __TwigTemplate_48c0b1feefc180b5f5157b2b62eb90aa->doDisplay(Array, Array)
#24 /var/www/html/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#25 /var/www/html/storage/runtime/compiled_templates/e9/e9e43b53633d303a365402ef4a162fe3.php(79): Twig\Template->display(Array, Array)
#26 /var/www/html/vendor/twig/twig/src/Template.php(394): __TwigTemplate_bf8bbf369c7a92c233be3ed10e0cecfb->doDisplay(Array, Array)
#27 /var/www/html/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#28 /var/www/html/vendor/twig/twig/src/Template.php(379): Twig\Template->display(Array)
#29 /var/www/html/vendor/twig/twig/src/TemplateWrapper.php(40): Twig\Template->render(Array, Array)
#30 /var/www/html/vendor/twig/twig/src/Environment.php(277): Twig\TemplateWrapper->render(Array)
#31 /var/www/html/vendor/craftcms/cms/src/web/View.php(465): Twig\Environment->render('feed-me/feeds/_...', Array)
#32 /var/www/html/vendor/craftcms/cms/src/web/View.php(518): craft\web\View->renderTemplate('feed-me/feeds/_...', Array)
#33 /var/www/html/vendor/craftcms/cms/src/web/TemplateResponseFormatter.php(56): craft\web\View->renderPageTemplate('feed-me/feeds/_...', Array, 'cp')
#34 /var/www/html/vendor/yiisoft/yii2/web/Response.php(1098): craft\web\TemplateResponseFormatter->format(Object(craft\web\Response))
#35 /var/www/html/vendor/craftcms/cms/src/web/Response.php(286): yii\web\Response->prepare()
#36 /var/www/html/vendor/yiisoft/yii2/web/Response.php(339): craft\web\Response->prepare()
#37 /var/www/html/vendor/yiisoft/yii2/base/Application.php(390): yii\web\Response->send()
#38 /var/www/html/web/index.php(12): yii\base\Application->run()
#39 {main}
```

### Steps to reproduce

1. Set up a form with repeater field.
2. Use the Feed-Me plugin to import submissions.
3. Proceed to the third step of the import process.

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

4.4.14

### Plugin version

2.0.31

### Multi-site?

Yes

### Additional context

The repeater field that triggers the error during the setup of the submission import includes a date field and radio buttons.

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.0.32"`.

**Comment by bavocorteville** [comment]
The fix indeed resolves the issue. However, I noticed that the date field is left empty, and it is not possible to import multiple repeater rows (via CSV?).

**Comment by engram-design** [comment]
That'd be correct at the moment (handling multiple rows) - at least for a CSV. It's difficult to break out multiple lines (which represent a submission) and multiple sub-lines (for Repeater rows) using the CSV format. XML or JSON would be fine. Related to https://github.com/craftcms/feed-me/issues/631 although this _should_ be addressed with the `csvColumnDelimiter` Feed Me [config setting](https://docs.craftcms.com/feed-me/v4/get-started/configuration.html)

As for the date field being empty, what value are you using? It'll just need to be a timestamp.

**Comment by bavocorteville** [comment]
I forgot that the option existed to export the submissions as JSON or XML. My bad. 
Regarding the date field, it was exported as dd/mm/yyyy.

**Comment by engram-design** [comment]
Fixed in [2.0.33](https://github.com/verbb/formie/releases/tag/2.0.33)
